i18zh-cn: Upgrade & Migrate -> Upgrade from Tauri 1.0

Wind-Explorer opened the pull request:
<!-- Thank you for opening a PR! We really appreciate you taking the time to help out 🙌  you're welcome -->

#### What kind of changes does this PR include?
<!-- Delete any that don’t apply -->
<!-- Translators, try to follow this pattern when naming your PRs:
"i18(language code): (short description)" -->

- Translated content for Tauri 2.0 docs

#### Description
- zh-cn translation of Upgrade from Tauri 1.0 page under Upgrade & Migrate page.

<!--
Here’s what will happen next:
1. Our GitHub bots will run to check your changes.
   If they spot any broken links you will see some error messages on this PR.
   Don’t hesitate to ask any questions if you’re not sure what these mean!
2. In a few minutes, you’ll be able to see a preview of your changes on Netlify 🥳
3. One or more of our maintainers will take a look and may ask you to make changes.
   We try to be responsive, but don’t worry if this takes a day or two.
4. Reach out to us on Discord with any questions along the way: 
   https://discord.com/invite/tauri
-->

Wind-Explorer committed:
i18zh-cn: Upgrade & Migrate -> Upgrade from Tauri 1.0

Wind-Explorer committed:
FIxed `#` links

Wind-Explorer committed:
Fix `#` links pt.2

lorenzolewis commented:
Thanks for this! @dklassic would you be able to help out with a review on this one?

dklassic commented on src/content/docs/zh-CN/guides/upgrade-migrate/from-tauri-1.mdx:
````diff
@@ -0,0 +1,1150 @@
+    })
+```
+
+`tauri::Builder::menu` now takes a closure because the menu needs a Manager instance to be built. See [the documentation](https://docs.rs/tauri/2.0.0-alpha/tauri/struct.Builder.html#method.menu) for more information.
+
+`tauri::Builder::menu` 现在接受一个闭包，因为菜单需要构建一个 Manager 实例。 有关更多信息，请参阅[文档](https://docs.rs/tauri/2.0.0-alpha/tauri/struct.Builder.html#method.menu)。
````
```suggestion
`tauri::Builder::menu` 现在接受一个闭包，因为菜单需要构建一个 Manager 实例。 有关更多信息，请参阅[文档](https://docs.rs/tauri/2.0.0-alpha/tauri/struct.Builder.html#method.menu)。
```

dklassic reviewed (changes requested):
Thanks for contributing!

I find the translation to be pretty good, but besides on particular direct change suggestion, I'd like to ask you to do another sweep to improve the reading experience:

- Ensure a space is in between Chinese characters and half sized characters.
- Remove redundant space between two full sized characters (both Chinese characters and punctuations).
- Properly replace half sized punctuation with full sized Chinese punctuations, e.g. use "：" instead of ": ".

Also regarding inline translation of technical term coexisting with the original term, for example in the case of "迁移 Path 路径 至 Tauri Manager 管理器" the formatting is rather confusing to read. I would suggest to use brackets to the mark translated term, i.e. "迁移 Path（路径）至 Tauri Manager（管理器）"

simonhyll committed:
Merge branch 'v2' into next

Wind-Explorer committed:
Merge branch 'tauri-apps:v2' into next

Wind-Explorer committed:
Update src/content/docs/zh-CN/guides/upgrade-migrate/from-tauri-1.mdx

Co-authored-by: DK Liao <dklassic@gmail.com>

Wind-Explorer committed:
More translation (see description)

Addressed points brought up by @dklassic, polished some improvable translations and translated new content.

Wind-Explorer committed:
Fix links (sacrificed special translation)

Wind-Explorer committed:
Fix links pt.2

Wind-Explorer commented:
Seems I'm a bit late to the party, translations for this particular page has been completed by #1967. Closing.
